cristin-project/src/main/java/no/unit/nva/cristin/projects/update/ProjectPatchValidator.java

```diff
@@ -243,8 +242,8 @@ private BadRequestException notAListOfMapsException() {
 
     private void validateWebPage(ObjectNode input) throws BadRequestException {
         if (input.has(WEB_PAGE) && !input.get(WEB_PAGE).isNull()) {
-            var webPage = input.get(WEB_PAGE).asText();
```

Comment:
Sometimes, its good to declare a local variable with a descriptive name for readability. Other times it just clutters the code and adds unnecessary overhead. So im drawn between using it or not using it.